src/storage/metrics.rs

```diff
@@ -25,6 +25,30 @@ make_static_metric! {
     pub struct SchedDurationVec: Histogram {
         "type" => RawReadKind,
     }
+
+    pub label_enum KvCommandKind {
+        prewrite,
+        commit,
+        cleanup,
+        rollback,
+        scan_lock,
+        resolve_lock,
+        gc,
+        unsafe_destroy_range,
+        delete_range,
+        pause,
+        key_mvcc,
+        start_ts_mvcc,
+        raw_put,
+        raw_batch_put,
+        raw_delete,
+        raw_delete_range,
+        raw_batch_delete,
+    }
+
+    pub struct KvCommandCounterVec: IntCounter {
```

Comment:
I believe that the enum of KvCommandCounterVec should be the same as SchedDurationVec?

Reply:
OK, I have change this.